src/qilib/configuration_helper/instrument_adapter.py

```diff
@@ -100,3 +100,11 @@ def accept(self, visitor: Visitor) -> None:
 
         """
         visitor.visit(self)
+
+    def __str__(self):
+        """Returns string representation for the InstrumentAdapter.
```

Comment:
```suggestion
        """ Returns string representation for the InstrumentAdapter.
```
But is a matter of taste I guess.

Reply:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

According to the only examples that we have, it should be without the space